_source/user-guide/infrastructure-monitoring/p8s-remote-write.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: article
+title: Remote write for Prometheus
+permalink: /user-guide/metrics/p8s-remote-write.html
+flags:
+  logzio-plan: community
+tags:
+  - metrics integrations
+contributors:
+  - yberlinger
+---
+This topic is for Prometheus users migrating their solution to Logz.io Infrastructure Monitoring powered by Grafana.
+
+You can send your Prometheus metrics to your Logz.io Infrastructure Monitoring account.
+To do this, you’ll need to send your application metrics to Logz.io in JSON format and stick to the formatting guidelines.
+
+Our log scraping software service takes all your log file data, scrapes it and stores it in an archive. The log data may include messaging, calendaring, and other data types. 
+
+Before you begin, make sure you have the following details for your Logz.io Metrics account: Listener address (URL) and Logz.io metrics account token.
+
```

Comment:
maybe also add explanations or links where he can find this data

Reply:
Done